### PR TITLE
[lexical-react] Feature: add attributes in LinkPlugin

### PR DESCRIPTION
**Pull request by taro-shono**
## Description

Add attributes when pasting a link into a text.

### Before

Can not add `target`

https://github.com/user-attachments/assets/175b85a0-8993-42b7-aa4b-7e0325536fe3

### After

Can add `target`

https://github.com/user-attachments/assets/c1a67866-8951-4f9d-acdf-c1750c4bd5eb

### CODE REVIEW

**Comment by potatowagon** [comment]
code lgtm, can we have some tests please to check that attributes are added to link on paste?

File: packages/lexical-react/src/LexicalLinkPlugin.ts [hunk 1]
```diff
@@ -38,13 +44,18 @@ export function LinkPlugin({validateUrl}: Props): null {
...
             }
             return false;
           } else {
             const {url, target, rel, title} = payload;
-            $toggleLink(url, {rel, target, title});
+            $toggleLink(url, {
+              ...attributes,
+              rel,
+              target,
+              title,
```
Comment (potatowagon) [inline packages/lexical-react/src/LexicalLinkPlugin.ts:57]: this means that the link attribute contents of payload will overwrite the contents of attributes

**Review by potatowagon — DISMISSED** [review]

**Comment by taro-shono** [comment]
@potatowagon I have added a test, so please check it out.

**Review by potatowagon — APPROVED** [review]

**Review by potatowagon — APPROVED** [review]

**Comment by potatowagon** [comment]
thanks for this!

**Comment by aleksey-pro** [comment]
Thanks for plugin. But if i switch off attributes after link is created - it doesn't removes target="__blank" atribute from node.

**Pull request merged**